server/leafnode.go

```diff
@@ -1764,6 +1765,13 @@ func (c *client) processLeafNodeConnect(s *Server, arg []byte, lang string) erro
 		return err
 	}
 
+	// Reject a cluster that contains spaces or line breaks.
+	if proto.Cluster != _EMPTY_ && (strings.Contains(proto.Cluster, " ") || strings.Contains(proto.Cluster, "\n")) {
```

Comment:
@ripienaar yes actually, updated here

Reply:
Would strings.ContainsAny() be faster/better?

Reply:
yeah like that better, updated